pkgs/servers/web-apps/shaarli/material-theme.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "shaarli-material-${version}";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "kalvn";
+    repo = "Shaarli-Material";
+    rev = "v${version}";
+    sha256 = "0ivq35183r5vyzvf47sgxwdxllmvhd5w9w75xgyp3kbw2na4yrmr";
+  };
+
+  patchPhase = ''
+    for f in material/*.html
+    do
+      substituteInPlace $f \
+        --replace '.min.css"' '.min.css#"' \
```

Comment:
With the `#`, the template engine used by Shaarli doesn't construct an absolute URL to the resource. Obviously nginx doesn't serve `http://your.domain/nix/store/…/`, so without the `#` the CSS and JS files don't get loaded.